### PR TITLE
ppo debugging

### PR DESCRIPTION
**Pull request by codekansas**

### CODE REVIEW

File: ksim/task/rl.py [hunk 1]
```diff
@@ -579,6 +579,10 @@ def log_reward_stats(self, transitions: Transition, reward_generators: Collectio
         for key, value in reward_stats.items():
             self.logger.log_scalar(key=key, value=value, namespace="reward")
 
+        # Logs the total reward.
+        total_reward = jnp.sum(transitions.reward) / num_episodes
```
Comment (budzianowski) [inline ksim/task/rl.py:580]: Common normalization is to scale with time and clip for long horizon.
```suggestion
        jnp.clip(rewards * self.config.ctrl_dt, 0, 1000)
```

Comment (codekansas) [reply ksim/task/rl.py:580]: would rather not do this
